problems/arrayMethods.js

```diff
@@ -130,7 +206,13 @@ const divisibleBy = () => {};
  * @returns {number[]} Numbers times their index
  */
 
-const numberTimesIdx = () => {};
+const numberTimesIdx = (nums) => {
+  return nums.map((el) => {
```

Comment:
Remember the callbacks take in the index too. 
(el, i) 
don't use indexOf later that destroys your time complexity when you already have the index.